index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = function loader(source) {
   return `
-    import { css } from 'lit-element';
+    import { css } from 'lit-element/lit-element.js';
```

Comment:
I guess this works with LitElement 2 and 3 (and Lit 2) but will (at least in [some cases](https://pnpm.io/faq#pnpm-does-not-work-with-your-project-here), e.g. when using pnpm) require the user project to have an explicit dependency to `lit-element` package which is something you might want to avoid (for simplicity) if you're using Lit 2 and already have a dependency to `lit`.

Would it make sense (and is it possible?) to add some loader option which would let the user configure the import path for `css`? Would be nice if you could have a loader option to set the import path to `'lit'` if you're using Lit 2. Or alternatively there could be a new version of `lit-css-loader` which would only be compatible with Lit 2 and using only the `lit` import but having the option to configure the path might be nice anyway.